mypy/typeanal.py

```diff
@@ -63,8 +64,9 @@ def analyze_type_alias(node: Expression,
                        allow_unnormalized: bool = False,
                        in_dynamic_func: bool = False,
                        global_scope: bool = True,
-                       warn_bound_tvar: bool = False) -> Optional[Type]:
-    """Return type if node is valid as a type alias rvalue.
+                       warn_bound_tvar: bool = False) -> Optional[Tuple[Type, Set[str]]]:
+    """If `node` is valid as a type alias rvalue, return the resulting type and a set of
```

Comment:
Style nit: first line of a multi-line docstring should be one line, followed by an empty line.